phivk/gradu.al
# Updates/additional graph data

Adds new metadata to nodes:

- Ongoing count of learner/sharers for each skill
- Submitted at attribute for each learner
- SubmittedBy for each skill based on when inititally created.

To test:
- Run the build step
- Check the outputted data
- Confirm visualisation still works.
